CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ## Bug Fixes:
 
+- Fix "TypeError: issubclass() arg 1 must be a class" When use Optional[Types] by [@lingfengchencn](https://github.com/lingfengchencn) in [PR 4200](https://github.com/gradio-app/gradio/pull/4200). 
+- Gradio will no longer send any analytics if analytics are disabled with the GRADIO_ANALYTICS_ENABLED environment variable. By [@akx](https://github.com/akx) in [PR 4194](https://github.com/gradio-app/gradio/pull/4194)
```

Comment:
Were analytics being sent when the env variable was set to false before this PR?

Reply:
What we call analytics weren't being sent, but network requests were being made to the Gradio server, see https://github.com/gradio-app/gradio/pull/4194#issuecomment-1546937585